js_test.go

```diff
@@ -939,6 +941,21 @@ func TestJetStreamExpiredPullRequests(t *testing.T) {
 			t.Fatalf("Expected error and wait for 250ms, got err=%v and dur=%v", err, dur)
 		}
 	}
+
+	t.Run("MaxWait timeout should return nats error", func(t *testing.T) {
+		_, err := sub.Fetch(1, MaxWait(12*time.Nanosecond))
+		if !errors.Is(err, ErrTimeout) {
+			t.Fatalf("Expect ErrTimeout, got err=%#v", err)
+		}
+	})
+	t.Run("Context timeout should return context error", func(t *testing.T) {
```

Comment:
I think this test case already covered here: https://github.com/nats-io/nats.go/blob/main/test/js_test.go#L6719-L6726

Reply:
:D